libs/datamodel/connectors/sql-datamodel-connector/src/postgres_datamodel_connector.rs

```diff
@@ -22,9 +49,58 @@ impl PostgresDatamodelConnector {
             ConnectorCapability::AutoIncrementNonIndexedAllowed,
         ];
 
-        let bigint = NativeTypeConstructor::without_args(BIG_INT_TYPE_NAME, ScalarType::Int);
+        let small_int = NativeTypeConstructor::without_args(SMALL_INT_TYPE_NAME, ScalarType::Int);
+        let integer = NativeTypeConstructor::without_args(INTEGER_TYPE_NAME, ScalarType::Int);
+        let big_int = NativeTypeConstructor::without_args(BIG_INT_TYPE_NAME, ScalarType::Int);
+        // let decimal = NativeTypeConstructor::with_args(DECIMAL_TYPE_NAME, 2, ScalarType::Decimal);
+        // let numeric = NativeTypeConstructor::with_args(NUMERIC_TYPE_NAME, 2, ScalarType::Decimal);
+        let real = NativeTypeConstructor::without_args(REAL_TYPE_NAME, ScalarType::Float);
+        let double_precision = NativeTypeConstructor::without_args(DOUBLE_PRECISION_TYPE_NAME, ScalarType::Float);
+        let small_serial = NativeTypeConstructor::without_args(SMALL_SERIAL_TYPE_NAME, ScalarType::Int);
+        let big_serial = NativeTypeConstructor::without_args(BIG_SERIAL_TYPE_NAME, ScalarType::Int);
         let varchar = NativeTypeConstructor::with_args(VARCHAR_TYPE_NAME, 1, ScalarType::String);
-        let constructors = vec![varchar, bigint];
+        let char = NativeTypeConstructor::with_args(CHAR_TYPE_NAME, 1, ScalarType::String);
+        let text = NativeTypeConstructor::without_args(TEXT_TYPE_NAME, ScalarType::String);
+        // let byte_a = NativeTypeConstructor::without_args(BYTE_A_TYPE_NAME, ScalarType::Bytes);
+        let timestamp = NativeTypeConstructor::with_args(TIMESTAMP_TYPE_NAME, 1, ScalarType::DateTime);
+        let timestamp_with_timezone =
+            NativeTypeConstructor::with_args(TIMESTAMP_WITH_TIMEZONE_TYPE_NAME, 1, ScalarType::DateTime);
```

Comment:
The datatype is a 2-element array of timestamps with timezones. I'm not sure how to represent that correctly here 🙏 

```suggestion
        let timestamp_with_timezone =
            NativeTypeConstructor::with_args(TIMESTAMP_WITH_TIMEZONE_TYPE_NAME, 1, ScalarType::DateTime);
        let timestamp_with_timezone_range =
            NativeTypeConstructor::with_args(TIMESTAMP_WITH_TIMEZONE_RANGE_TYPE_NAME, 1, [ScalarType::DateTime, ScalarType::DateTime]); // Needs to be a 2-element array

```